packages/twenty-docker/k8s/manifests/deployment-server.yaml

```diff
@@ -39,6 +43,8 @@ spec:
           value: "true"
         - name: STORAGE_TYPE
           value: "local"
+        - name: "MESSAGE_QUEUE_TYPE"
+          value: "pg-boss"
```

Comment:
LGTM for now but we are aiming for bull-mq (this needs a redis to be setup) setup long term which is more robust